roles/ispmail-rspamd/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Add rspamd APT signing key
+  apt_key:
```

Comment:
This won't pass ansible-lint 4.0.0 new rules, we should add something like:
```yaml
  register: apt_key_result
  retries: 3
  until: apt_key_result is succeeded
```

Reply:
done